rdm/extensions.py

```diff
@@ -0,0 +1,66 @@
+from importlib import import_module
+
+from jinja2 import nodes
+from jinja2.ext import Extension
+
+
+class RdmExtension(Extension):
```

Comment:
At first look, it seems like most of the code in here could be moved to `render.py`.

It would be slightly more convenient for users if we could avoid needing a separate class for this.  I.e., if `render.py` could look at the extensions to see if there was a `postprocess` method, and call it.

If this isn't possible (or there are other reasons to keep a subclass of `Extension`), then maybe `PostProcessingExtension` would be more appropriate than `RdmExtension`?